cmd/livepeer_cli/wizard_bond.go

```diff
@@ -329,5 +329,15 @@ func (w *wizard) withdrawStake() {
 }
 
 func (w *wizard) withdrawFees() {
-	httpPost(fmt.Sprintf("http://%v:%v/withdrawFees", w.host, w.httpPort))
+	dInfo, err := w.getDelegatorInfo()
+	if err != nil {
+		glog.Errorf("Error getting delegator info: %v", err)
+		return
+	}
+
+	val := url.Values{
+		"amount": {fmt.Sprintf("%v", dInfo.Fees.String())},
```

Comment:
```suggestion
		"amount": {fmt.Sprintf("%v", dInfo.PendingFees.String())},
```

If an account has earned fees in recent round, then the `PendingFees` field [1] on the `lpTypes.Delegator` struct can be greater than the `Fees` field. This is because within the BondingManager contract each address has a `fees` state that is only updated when the address submits a transaction to the contract and when that happens the `fees` state is incremented by any additional fees that have been earned recently. Prior to this, you can calculate the fees for an address, which includes its current `fees` state plus any additional earned, by calling the `pendingFees()` function on the BondingManager contract [2]. 

So, we should use the `PendingFees` field here because it will represent the address' true total fees that can be withdrawn while the `Fees` field *might* be missing some fees if the address' `fees` state in the contract hasn't been updated in awhile.

Note: The same relationship exists between the `BondedAmount` and `PendingStake` fields with the BondingManager contract function `pendingStake()` calculating the address' current stake.

[1] https://github.com/livepeer/go-livepeer/blob/81a42873663bd34cbba007cb11a5554f84404f92/eth/client.go#L699
[2] https://github.com/livepeer/go-livepeer/blob/81a42873663bd34cbba007cb11a5554f84404f92/eth/client.go#L679

Reply:
Ha, good catch! Thanks for this comment.